src/D2L.CodeStyle.TestAnalyzers/NUnit/CategoryAnalyzer.cs

```diff
@@ -189,11 +193,16 @@ ISymbol symbol
 
 			TypedConstant arg = args[0];
 
+			if( arg.Type == null ) {
+				return;
+			}
+
 			if( arg.Type.SpecialType != SpecialType.System_String ) {
 				return;
 			}
 
-			string category = arg.Value as string;
+			string category = (string?)arg.Value!;

```

Comment:
We know that arg.Type is System.String. I'm not sure if `null` can actually be returned here, I somewhat doubt it -- but this is equivalent behaviour fwiw

Reply:
Does `[TestCategory( null )]` bail out on an earlier check? I guess it's treated as `object` because the constant is evaluated independently of the context it's used in?

Reply:
I'm not sure